ui/app/scripts/controllers/create.js

```diff
@@ -132,12 +132,17 @@
         statusChanged: function() {
           model.demo.demoStatus.lastStatusTimestamp = new Date().toJSON();
         },
-        deleteAttachment: function(attachent, index) {
-          demoService.deleteAttachment($routeParams.uri, attachent).then(
+        deleteAttachment: function(attachment, index) {
+          demoService.deleteAttachment($routeParams.uri, attachment).then(
             function() {
               model.demo.attachments.splice(index, 1);
             }
           );
+          for (var i = 0; i < model.demo.memos.length; i++) {
+              if (model.demo.memos[i].title === attachment.attachmentName) {
+                model.demo.memos.splice(i, 1);
+              }
+          }
```

Comment:
Does this get pushed to the server? I thought not, but on second thought, this is create controller..